acqa-repo1-aws-tf12-part1.tf

```diff
@@ -458,10 +485,10 @@ resource "aws_autoscaling_group" "acqa-test-asg1" {
   vpc_zone_identifier       = [aws_subnet.acqa-test-albsubnet1.id, aws_subnet.acqa-test-albsubnet2.id]
 
   initial_lifecycle_hook {
-    name                 = "aqa-test-asg1-lifecyclehook1"
-    default_result       = "CONTINUE"
-    heartbeat_timeout    = 2000
-    lifecycle_transition = "autoscaling:EC2_INSTANCE_LAUNCHING"
+    name                  = "aqa-test-asg1-lifecyclehook1"
+    default_result        = "CONTINUE"
+    heartbeat_timeout     = 2000
+    lifecycle_transition  = "autoscaling:EC2_INSTANCE_LAUNCHING"
     notification_metadata = <<EOF
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure point-in-time-recovery (PITR) is enabled for AWS DynamoDB tables**
 
**Vulnerability**
AWS DynamoDb tables have point-in-time-recovery (PITR) disabled.
 
**Remediation**
AWS offers On-Demand Backup & Restore and Continuous backup options through Point-In-Time-Recovery. Point-in-time recovery helps protect your DynamoDB tables from accidental write or delete operations. With point-in-time recovery, you don't have to worry about creating, maintaining, or scheduling on-demand backups. Enable Point-In-Time-Recovery in your AWS DynamoDB instance.

source: accurics-